lib/google_scraping/account/schemas/user.ex

```diff
@@ -45,7 +46,9 @@ defmodule GoogleScraping.Accounts.Schemas.User do
   defp validate_email(changeset) do
     changeset
     |> validate_required([:email])
-    |> validate_format(:email, ~r/^[^\s]+@[^\s]+$/, message: "must have the @ sign and no spaces")
+    |> validate_format(:email, ~r/^[^\s]+@[^\s]+$/,
```

Comment:
How about using module attrubute for the email regex, so that we don't have to hardcoded here?

```elixir
# At the top
@email_regex  ~r/^[^\s]+@[^\s]+$

# Then here
|> validate_format(:email, @email_regex, ...
```